tests/test_wsgi_servers.py

```diff
@@ -232,3 +239,43 @@ def test_static_file(self, server_url):
         assert resp.headers.get('Content-Disposition') == (
             'attachment; filename="test_wsgi_servers.py"'
         )
+
+        content_length = int(resp.headers['Content-Length'])
+        file_size = os.path.getsize(__file__)
+        assert len(resp.content) == content_length == file_size
+
+    @pytest.mark.parametrize(
+        'byte_range,expected_head',
+        [
+            ('7-', b'hashlib'),
+            ('2-6', b'port'),
+            ('32-38', b'random'),
+            ('-47', b'The content of this comment is part of a test.\n'),
```

Comment:
I'm -1 on using this file to test stuff, since each time we change something this test will break for no particular reason.

Let's create a sample file we can use so that we avoid that issue

Reply:
Right... hopefully you could accept it this time, because apart from that last comment, this was already in place for testing static files (see the above test), which you accepted on review before.

Reply:
I guess we could leave a `# TODO` comment for this?

Reply:
Will add.

Reply:
thanks